references/cli/addon.go

```diff
@@ -314,6 +362,7 @@ func listAddons(ctx context.Context, clt client.Client, registry string) error {
 	if err != nil {
 		return err
 	}
+	onlineAddon := map[string]bool{}
```

Comment:
If an add-on registry is valid, it's online?

Reply:
Just a local variable in this func， won't show this info for user.